docker/crontab

```diff
@@ -1,9 +1,9 @@
-PATH=/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin
+PATH=/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/code/listenbrainz/admin
 PYTHONPATH=/usr/local/lib/python36.zip:/usr/local/lib/python3.6:/usr/local/lib/python3.6/lib-dynload:/usr/local/lib/python3.6/site-packages:/usr/local/lib/python3.6/site-packages/messybrainz-99.0.0.dev0-py3.6.egg
 GOOGLE_APPLICATION_CREDENTIALS=/code/listenbrainz/credentials/bigquery-credentials.json
 
 # Statistics calculation via Google BigQuery
 0 0 * * 0 /usr/local/bin/python /code/listenbrainz/manage.py stats calculate >> /var/log/stats.log 2>&1
 
 # Data dumps creation for backup and public use
-15 0 1 * * /usr/local/bin/python /code/listenbrainz/manage.py dump create >> /var/log/dump_create.log 2>&1
+15 0 1 * * RunExport >> /var/log/dump_create.log 2>&1
```

Comment:
For the sake of being explicit, which is a good thing in cron jobs, I think we should use exact paths to the binaries. if that allows us to get rid of the PATH statement, even better.